src/nmo.js

```diff
@@ -34,10 +42,7 @@ Object.defineProperty(nmo, 'cli', {
   }
 });
 
-nmo.version = pkg.version;
-
-const commandFuncs = {}, cliFuncs = {};
-nmo.load = function load (opts) {
+nmo.load = function (opts) {
```

Comment:
the now anonymous function was named for a reason, so you could identify it easily in stacktraces / a dump of the JavaScript heap (vs <anonymous function> or <closure> for an anonymous function)